astronomer/providers/google/cloud/operators/kubernetes_engine.py

```diff
@@ -96,5 +99,16 @@ def execute(self, context: "Context") -> None:
             location=self.location,
             use_internal_ip=self.use_internal_ip,
         ) as config_file:
+            with open(config_file) as tmp_file:
+                self.config_file_cache = yaml.load(tmp_file, Loader=yaml.SafeLoader)
+
             self.config_file = config_file
             return super().execute(context)
+
+    def trigger_reentry(self, context: Context, event: Dict[str, Any]) -> Any:
+        """Point of re-entry from trigger."""
+        with tempfile.NamedTemporaryFile() as conf_file:
+            yaml.dump(self.config_file_cache, conf_file)
```

Comment:
Once control will resume from the trigger `self.config_file_cache`  become None

Reply:
I made a little fix , very hacky , let me know what yo think